extensions.lua

```diff
@@ -152,6 +153,20 @@ local function process_config(conf)
         end
 
         for _, event in ipairs(fconf.events) do
+            if event.http == nil then
+                -- do nothing
+            elseif ret.exports[event.http.path] ~= nil then
+                    return nil, ExtensionConfigError:new(
+                        "Invalid extensions config: " ..
+                        "collision of http event '%s'" ..
+                        " to handle function '%s'",
+                        event.http.path, fname
+                    )
+            else
+                httpd:route({ path = event.http.path, method = event.http.method}, fn)
+                ret.exports[event.http.path] = fn
+            end
```

Comment:
What about removing the endpoint from the config? It should be implemented and tested.